engine.py

```diff
@@ -7,12 +7,15 @@
 """
 import os
 import sys
-import ctypes
-import shutil
-import logging
 import traceback
 
+from PyQt4 import QtGui
+
+import sgtk
```

Comment:
I do not want to initialize the engine, that's the whole point of hijacking the context before